### ISSUE TITLE
Unable to query input and output devices on 2.25.x+

### ISSUE THREAD

**Issue by Cereal-Killa** [opentok/opentok-windows-sdk-samples#61]
**Describe the bug**
Up to version 2.24.2 getting input and output devices works no problem, 2.25.0 onwards returns an empty array.

**To Reproduce**
Create a WPF app and run the following code:
var Devices = OpenTok.AudioDevice.EnumerateInputAudioDevices();

**Expected behavior**
Get an array of the input audio devices in the system.

**Screenshots**
2.25.0 and higher:
![image](https://github.com/opentok/opentok-windows-sdk-samples/assets/25939765/09eca3ac-4be5-4f8a-9014-9bdfd70d056a)

2.24.2 and lower:
![image](https://github.com/opentok/opentok-windows-sdk-samples/assets/25939765/8bea7a0c-518f-4cfd-906e-114ff2874bcd)

**Device (please compete the following information):**
- Windows SDK version: 2.25.3
- OS and version: Windows 10 22H2

**Additional context**
When building the app as x64 the app runs without any exception, but won't list the devices unless I downgrade to 2.24.2. Running the app as x86 or "Any Cpu" throws "An attempt was made to load a program with an incorrect format". (unrelated but are there dlls for those binaries?)

**Comment by sergioturgil-vonage** [comment]
Hi, the problem is real and the fix is coming in 2.26.1

To clarify, the origin of the problem is a change in the implementation where we started using the internal webrtc audio module to enumerate devices instead of a platform specific way.
This adds the requirement for webrtc to be initialized before enumerating, which generally happens when you create the first publisher. This also means that moving enumeration after publisher creation would also fix the problem if waiting for 2.26.1 is not an option.

We had to make this change as part of an effort to make the SDK compatible with platforms that might not have that platform specific implementation of audio device enumeration.
